Essentials/src/main/java/com/earth2me/essentials/signs/SignTrade.java

```diff
@@ -268,37 +272,44 @@ protected final Trade getTrade(final ISign sign, final int index, final AmountTy
     }
 
     protected final void subtractAmount(final ISign sign, final int index, final Trade trade, final IEssentials ess) throws SignException {
+        subtractAmount(sign, index, trade, ess, false);
```

Comment:
Is it worth removing the overloads so it's clear at the call site whether we're calling this for validation or for real?